oe_theme.theme

```diff
@@ -1522,3 +1523,88 @@ function oe_theme_preprocess_page(array &$variables): void {
     'id' => ['main-content'],
   ]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables): void {
```

Comment:
This is out of scope, the current ticket is supposed to only provide the patter, not to style the organization entities. We will do this when needed as it will need to be paired by a test too. Please move out of this PR everything concerning this, keep it on a separate branch or something, as it might be useful later on (I've left some comments anyway before realizing that it was out of scope).